### PR TITLE
[zh] sync/blog/_posts/2022-09-21-csi-nodeexpandsecret.md

### PR DESCRIPTION
**Pull request by liangyongzhenya**

### CODE REVIEW

**Comment by windsonsea** [comment]
Please read carefully the [localization style guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/) through, including but not limited to:
- 长句折行
- 注释的段落 10 行为佳，否则不方便 review
- 不要忽视 review comments
- 不要轻易 close 收到 comments 的 PR，努力完成它，你才会有进步

**Comment by Sea-n** [comment]
Hey @liangyongzhenya,

It's been a while since the last update, would you like to revise your PR so that we can move forward?

File: content/zh-cn/blog/_posts/2022-09-21-csi-nodeexpandsecret.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+Kubernetes v1.25, released earlier this month, introduced a new feature
+that lets your cluster expand storage volumes, even when access to those
+volumes requires a secret (for example: a credential for accessing a SAN fabric)
+to perform node expand operation. This new behavior is in alpha and you
+must enable a feature gate (`CSINodeExpandSecret`) to make use of it.
+You must also be using [CSI](https://kubernetes-csi.github.io/docs/)
+storage; this change isn't relevant to storage drivers that are built in to Kubernetes.
+-->
+
+本月初发布的 Kubernetes v1.25 引入了一项新功能让你的集群可以扩展存储卷，即使访问这些卷需要一个秘钥（例如：访问 SAN 结构的证书），也可以让你的集群扩展存储卷。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-21-csi-nodeexpandsecret.md:34]:
```suggestion
本月初发布的 Kubernetes v1.25 引入了一项新功能，让你的集群可以扩展存储卷，
即使访问这些卷时需要 Secret（例如：用于访问 SAN 存储的凭据）才能执行节点层面扩展操作也没问题。
```

File: content/zh-cn/blog/_posts/2022-09-21-csi-nodeexpandsecret.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+<!--
+To turn on this new, alpha feature, you enable the `CSINodeExpandSecret` feature
+gate for the kube-apiserver and kubelet, which turns on a mechanism to send `secretRef`
+configuration as part of NodeExpansion  by the CSI drivers thus make use of
+the same to perform node side expansion operation with the underlying
+storage system.
+-->
+
+要打开这个新的 Alpha 功能，你需要启用 `CSINodeExpandSecret` 功能启用 kube-apiserver 和 kubelet，该功能开启了发送 "secretRef " 配置的机制。
+作为 NodeExpansion 的一部分由 CSI 驱动发送，从而利用这些配置来执行节点侧的扩展操作与底层的存储系统。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-21-csi-nodeexpandsecret.md:47]:
```suggestion
从而利用相同的配置来针对下层的存储系统执行节点侧的扩展操作。
```

File: content/zh-cn/blog/_posts/2022-09-21-csi-nodeexpandsecret.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+Before Kubernetes v1.24, you were able to define a cluster-level StorageClass
+that made use of [StorageClass Secrets](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html),
+but you didn't have any mechanism to specify the credentials that would be used for
+operations that take place when the storage was mounted onto a node and when
+the volume has to be expanded at node side.
+-->
+
+### 这究竟是怎么回事？
+
+在 Kubernetes v1.24 之前，你可以定义一个集群级的 StorageClass 使用 [StorageClass Secrets](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-21-csi-nodeexpandsecret.md:61]:
```suggestion
在 Kubernetes v1.24 之前，你可以定义一个集群级的 StorageClass，在其中使用
[StorageClass Secrets](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html)。
```

**Review by tengqm — COMMENTED** [review]
/hold
for low-quality translation.

**Review by tengqm — COMMENTED** [review]
/hold
for low-quality translation.

**Comment by tengqm** [comment]
No responses from author for over a month. Closing.
/close

**Pull request closed without merging**